### PR TITLE
Solution jv-lottery task

### PR DESCRIPTION
**Pull request by MaxMeleshchenko**

### CODE REVIEW

**Review by Ivan95kos — CHANGES_REQUESTED** [review]
fix all checks
Знімок екрана 2023-02-20 о 22 00 48

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i]);
+        }
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Application.java:11]:
```suggestion
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```

File: src/main/java/core/basesyntax/Application.java [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
     }
 }
+
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Application.java:14]: remove empty line

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 3]
```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: move Random to the class field, like:
image

File: src/main/java/core/basesyntax/Lottery.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:8]: Move CollorSupplier and Random to the class field

File: src/main/java/core/basesyntax/Lottery.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        Ball ball = new Ball();
+
+        ball.setNumber(new Random().nextInt(100));
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:11]: 100 is magic number
image

File: src/main/java/core/basesyntax/Lottery.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:6]: remove empty line

**Review by kozhukhovsky — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Application.java (hunk 2 above)
Comment (MaxMeleshchenko) [reply src/main/java/core/basesyntax/Application.java:14]: Throws an error when deleting an empty line : [NewlineAtEndOfFile]

**Review by kozhukhovsky — APPROVED** [review]

**Pull request closed without merging**